ndr_mapi.c

```diff
@@ -448,6 +476,40 @@ _PUBLIC_ enum ndr_err_code ndr_pull_mapi2k7_AuxInfo(struct ndr_pull *ndr, int nd
 	return NDR_ERR_SUCCESS;
 }
 
+
+_PUBLIC_ enum ndr_err_code ndr_push_mapi2k7_AuxInfo(struct ndr_push *ndr, int ndr_flags, const struct mapi2k7_AuxInfo *r)
+{
+	uint32_t	i;
+	
```

Comment:
If you do a git diff, you will see something similar to the output below. All the red lines are extra spaces to be removed.
![space](https://cloud.githubusercontent.com/assets/3603061/6581915/41efbe30-c75a-11e4-937f-48270d886273.png)